deegree-core/deegree-core-featureinfo/src/main/java/org/deegree/featureinfo/serializing/GeoJsonFeatureInfoSerializer.java

```diff
@@ -0,0 +1,66 @@
+package org.deegree.featureinfo.serializing;
```

Comment:
This file is missing the necessary Java file header (see https://github.com/deegree/deegree3/wiki/Developer-guidelines#java-file-headers)

Reply:
Added header with https://github.com/deegree/deegree3/pull/1623/commits/39e39c1de94d59f8a026218a93ba07e3b66dd4d9